### PR TITLE
Add example for using parallel policies.

### LINKED ISSUE #11: Allow policies to run in series or parallel
It would be nice if we could emulate the functionality from hapi route prerequisites to indicate that policies are allowed to run in parallel or in series when they're part of the same request lifecycle extension.

See
  - [hapi code for this](https://github.com/hapijs/hapi/blob/93292ce5a53e6c51077fadabb59831bb53c3688c/lib/handler.js#L34)
  - [hapi syntax for this](https://github.com/hapijs/hapi/blob/7410b137259dce42ea6fef951a15e2bc612440ad/API.md#route-prerequisites)

### PR DESCRIPTION
**Pull request by devinivy**
Closes #11 .

### CODE REVIEW

**Comment by mark-bradshaw** [comment]
Did you forget the new policy files?  Seems incomplete.

**Comment by devinivy** [comment]
Oops!  That should be better.

File: example/policies/moonInPhase.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+var Wreck = require('wreck');
+var Boom  = require('boom');
+
+var moonInPhase = function (request, reply, next) {
+ 
+    var now = (new Date()).getTime();
```
Comment (mark-bradshaw) [inline example/policies/moonInPhase.js]: Could be Date.now()?

File: example/policies/moonInPhase.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+        if (payload.Phase !== 'Waning Crescent') {
+            
+            return next(null, false, payload[0].Phase + ' is the wrong moon phase for admin night.');
+        }
+
+        next(null, true);
+    });
+
+}
```
Comment (mark-bradshaw) [inline example/policies/moonInPhase.js]: Cool example. :)

File: example/policies/addAnalytics.js [hunk 3]
```diff
@@ -8,7 +8,6 @@ var addAnalytics = function (request, reply, callback) {
     callback(null, true);
 };
 
-addAnalytics.pre = false;
-addAnalytics.post = true;
+addAnalytics.applyPoint = 'onPostHandler';
```
Comment (mark-bradshaw) [inline example/policies/addAnalytics.js]: Thanks.

**Comment by mark-bradshaw** [comment]
Looks good.  Once you're happy please merge.  Night.

**Comment by devinivy** [comment]
:+1: Goodnight!

**Pull request merged**